README.md

```diff
@@ -5,7 +5,7 @@ This repo contains the source code for
 [Markdown](http://daringfireball.net/projects/markdown/syntax). If you're not familiar with it, fear not; it's easy
 to learn and to become rapidly productive!
 
-Everything you push into the gh-pages branch goes live immediately!
+Everything you push into the GH-pages branch goes live immediately!
```

Comment:
I don't think this is an improvement since `gh-pages` is the actual branch name.

Reply:
`gh-pages` would probably be a better way to refer to it, although I'm sure GH docs could shed some light on how they format things. I'll take a look.

Reply:
Yep. If we were to change to anything, it should be this `gh-pages` as per their style:
https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/unpublishing-a-github-pages-site#unpublishing-a-project-site